tests/test-repo.c

```diff
@@ -151,6 +151,56 @@ test_repo_equal (Fixture       *fixture,
   g_assert_false (ostree_repo_equal (closed_repo, closed_repo));
 }
 
+static void
+test_repo_get_min_free_space (Fixture *fixture,
+                              gconstpointer test_data)
+{
+  g_autoptr (GKeyFile) config = NULL;
+  g_autoptr(GError) error = NULL;
+  gint i = 0;
+  g_autoptr(OstreeRepo) repo = ostree_repo_create_at (fixture->tmpdir.fd, ".",
+                                                      OSTREE_REPO_MODE_ARCHIVE,
+                                                      NULL,
+                                                      NULL, &error);
+  g_assert_no_error (error);
+
+  const char *values_to_test[] = { "500MB",
+                                   "0MB",
+                                   "17179869185GB", /* Overflow parameter: bytes > G_MAXUINT64 */
+                                   NULL
+                                 };
+
+  config = ostree_repo_copy_config (repo);
+
+  while (values_to_test[i] != NULL)
```

Comment:
Nitpick: Use a `for` loop for this. That saves you having to manually increment `i` on every loop path below.

i.e. `for (i = 0; values_to_test[i] != NULL; i++)`